docs/sources/developers/plugins/_index.md

```diff
@@ -2,81 +2,52 @@
 aliases:
   - ../plugins/developing/
 description: Resources for creating Grafana plugins
-title: Build a plugin
+title: Welcome to the Grafana plugin developer's guide
 weight: 200
 ---
 
-# Build a plugin
+# Grafana plugin developer's guide
 
-For more information on the types of plugins you can build, refer to the [Plugin Overview]({{< relref "../../administration/plugin-management/" >}}).
+Grafana is an open source software project for creating, exploring, and sharing data through dashboards. Plugins extend Grafana's built-in functionality so that users can accomplish specialized tasks to make Grafana even more useful for them. 
```

Comment:
I probably wouldn't explain Grafana, even briefly, plugin development is a suitably advanced topic that the audience understands this already.

I would maybe include some examples of what plugins do as we have on the grafana intro, specialized tasks feels vague. Example from `/docs/grafana/latest/introduction/`:
> the Grafana OSS plugin framework also enables you to connect other data sources like NoSQL/SQL databases, ticketing tools like Jira or ServiceNow, and CI/CD tooling like GitLab.

Reply:
Fixed in latest commit.